android-components/components/feature/addons/src/test/java/mozilla/components/feature/addons/ui/AddonsManagerAdapterTest.kt

```diff
@@ -225,7 +225,6 @@ class AddonsManagerAdapterTest {
 
         whenever(titleView.context).thenReturn(testContext)
         whenever(summaryView.context).thenReturn(testContext)
-        whenever(iconContainer.context).thenReturn(testContext)
```

Comment:
Out of curiosity, why are we removing this?😜

Reply:
that doesn't appear to be used

Reply:
Thanks for the clarification!